dateparser_cli/utils.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import os
+from pathlib import Path
+
+DEFAULT_UNIX_CACHE_DIR = os.environ.get("DEFAULT_CACHE_DIR", '~/.cache')
+DEFAULT_WIXDOWS_CACHE_DIR = os.environ.get("DEFAULT_CACHE_DIR", os.path.join(str(Path.home()), "AppData", "Roaming"))
+DEFAULT_DIR_NAME = os.environ.get("DEFAULT_DIR_NAME", 'dateparser_models')
+
+dateparser_model_home = None
+_cache_dir = None
```

Comment:
Please, remove these two statements :)

In case of `_cache_dir`, as you are using an `if` with a broad `else`, it will always take a value, so you don't need to initialize it. On the other hand, in case of `dateparser_model_home` it is defined below, so the same: you don't need to initialize it.

Reply:
Thanks, Marc, I have removed this initialization